lib/rules/require-unicode-regexp.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
         type: "suggestion",
 
         docs: {
-            description: "Enforce the use of `u` or `v` flag on RegExp",
+            description: "Enforce the use of `u` or `v` flag on `RegExp`",
```

Comment:
```suggestion
            description: "Enforce the use of `u` or `v` flag on regular expressions",
```

It might be better like this, as this doesn't only refer to the `RegExp` constructor.